src/core_plugins/vega/public/vega_editor.template.html

```diff
@@ -0,0 +1,75 @@
+<div class="vegaEditor" ng-controller="VegaEditorController">
+  <div
+    ui-ace="{
+      onLoad: aceLoaded,
+      useWrapMode: true,
+      advanced: {
+        highlightActiveLine: false
+      },
+      rendererOptions: {
+        showPrintMargin: false
+      },
+      mode: 'hjson'
+    }"
+    id="vegaAceEditor"
+    ng-model="vis.params.spec"
+    data-test-subj="vega-editor"
+  ></div>
+
+  <div id="editor_actions">
+    <span dropdown>
+      <button
+        id="vegaHelp"
+        class="editor_action"
+        dropdown-toggle href="#"
+        aria-label="Vega help"
+      >
+        <span class="kuiIcon fa-question-circle"></span>
+      </button>
+      <ul
+        class="dropdown-menu"
+        role="menu"
+        aria-labelledby="vegaHelp"
+      >
+        <li role="menuitem">
+          <a target="_blank" href="https://github.com/nyurik/kibana-vega-vis#vega-visualization-plugin-for-kibana">
```

Comment:
this link will need to point to the documentation for Kibana, not the community plugin. It's fine to point to the latest master docs.

Like the drop-down btw, presents well.